games-arcade/opensyobonaction/opensyobonaction-2.rc2.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="An open-source cross-platform port of the Japanese platformer using SDL"
```

Comment:
The limit is 80 chars, haiku porter told it to me. I made it shorter because this.

> 2016. aug. 11. dátummal, 22:24 időpontban fbrosson <notifications@github.com> írta:
> 
> In games-arcade/opensyobonaction/opensyobonaction-2.rc2.recipe <https://github.com/haikuports/haikuports/pull/761#discussion_r74496121>:
> 
> > @@ -0,0 +1,75 @@
> > +SUMMARY="An open-source cross-platform port of the Japanese platformer using SDL"
> If I'm not mistaken, the line with the SUMMARY variable should be <= 70 chars.
> I don't remember, however, if that's the limit for the whole line or just for the content of the variable.
> I should have looked for the info, but I was affraid of comming back too late...
> I think the limit is for the whole line, not counting just the newline. If someone could confirm that would be great ;-)
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/haikuports/haikuports/pull/761/files/6b085cd3cfcd027be41c3ded31eae0e771e35018#r74496121>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AFT6I6rsjZ8vBCp1tO4rwD32mOE6tV0bks5qe4T4gaJpZM4JhjLd>.
>